src/back/spv/writer.rs

```diff
@@ -762,13 +762,16 @@ impl Writer {
     fn write_scalar(&self, id: Word, kind: crate::ScalarKind, width: u8) -> Instruction {
         match kind {
             crate::ScalarKind::Sint => {
-                self.instruction_type_int(id, (width * BITS_PER_BYTE) as u32, Signedness::Signed)
+                let bits = (width * BITS_PER_BYTE) as u32;
```

Comment:
we should move this outside of the `match`

Reply:
I wouldn't do that. I also believed that to be doable, but `crate:ScalarKind::Bool` doesn't use this. 

Question is: do we want to move it out, so we only have it once in the code, but let one match not use it. We can also choose to not move it, like this, but having 3 times the same variable.

Reply:
it's fine if one of the match arms don't use it